File: 2161-partition-array-according-to-given-pivot/2161-partition-array-according-to-given-pivot.js
/**
 * @param {number[]} nums
 * @param {number} pivot
 * @return {number[]}
 */
var pivotArray = function(nums, pivot) {
        let Larr=[];
    let Marr=[];
    let Rarr=[];
    
    for (let i = 0; i < nums.length; i++) {
        if(nums[i]>pivot){
            Larr.push(nums[i]);
        }else if(nums[i]==pivot){
            Marr.push(nums[i]);
        }else{
            Rarr.push(nums[i])
        }
    }
    return [...Rarr,...Marr,...Larr]
};

File: 2161-partition-array-according-to-given-pivot/2161-partition-array-according-to-given-pivot.cs
public class Solution {
    public int[] PivotArray(int[] nums, int pivot) {
        List<int> result = new List<int>();

        foreach (int num in nums) {
            if (num < pivot) {
                result.Add(num);
            }
        }

        foreach (int num in nums) {
            if (num == pivot) {
                result.Add(num);
            }
        }

        foreach (int num in nums) {
            if (num > pivot) {
                result.Add(num);
            }
        }

        return result.ToArray();
    }
}